trade-mapping-to-ipaffs/src/main/java/uk/gov/defra/stw/mapping/toipaffs/common/commodities/CommodityTemperatureMapper.java

```diff
@@ -15,7 +15,7 @@
 import uk.gov.defra.tracesx.notificationschema.representation.enumeration.CommodityTemperature;
 
 @Component
-public class ChedpTemperatureMapper implements Mapper<SpsCertificate, CommodityTemperature> {
+public class CommodityTemperatureMapper implements Mapper<SpsCertificate, CommodityTemperature> {
```

Comment:
Don't need "Commodity" on the front of the class name, the IPAFFS field this goes into is just called `temperature`